documentation/src/main/markdown/packagemgrs/cargo.md

```diff
@@ -1,11 +1,32 @@
 # Cargo support
 
-## Overview
+## Overview  
 
-[detect_product_short] runs the Cargo detector if it finds either of the following files in your project:
+[detect_product_short] now includes two Cargo detectors:
 
-* Cargo.toml
-* Cargo.lock
+* **Cargo CLI Detector** (New)  
+* **Cargo Lockfile Detector**  
 
-The Cargo detector parses the Cargo.lock file for information on your project's dependencies. If the detector discovers a Cargo.toml file but not a Cargo.lock file, it will prompt the user to generate a Cargo.lock by running `cargo generate-lockfile` and then run [detect_product_short] again.
-The Cargo detector extracts the project's name and version from the Cargo.toml file.  If it does not find a Cargo.toml file, it will defer to values derived by Git, from the project's directory, or defaults.
+## Cargo CLI Detector  
+
+A new cli detector for Cargo projects, which extracts **direct and transitive dependencies** using the `cargo tree` command. This improves the accuracy of dependency detection over the previous flat-list detection.
```

Comment:
Suggestion:
`Detector for Cargo projects, extracts **direct and transitive dependencies** using the `cargo tree` command. `